infra/app/dapr-state-postgres.bicep

```diff
@@ -11,15 +11,22 @@ param tags object = {
   'azd-env-name': name
 }
 
-module daprBindingPGResources '../core/postgres.bicep' = {
+module daprBindingPGResources '../core/flexibleserver.bicep' = {
   name: 'dapr-binding-pg-${resourceToken}'
   params:{
-    name: name
+    name: 'pg${resourceToken}'
     location: location
     tags: tags
-    postgresUser: postgresUser
-    postgresPassword: postgresPassword
+    sku: {
+      name: 'Standard_B2s'
```

Comment:
I usually use Standard_B1s with only 1 V Core, but you previously used 2 V Cores, so I upped it. Does it need higher? (Otherwise I think samples should go for cheapest/smallest, to avoid wasting resources/money)